fbpost/test_dir/test_post_methods.py

```diff
@@ -160,7 +187,58 @@ def test_react_to_post_user_corresponding_post_change_reaction():
 
 # test reaction metrics
 @pytest.mark.django_db
-def test_reaction_metrics_count_type(post_setup):
+def test_reaction_metrics_post_exists(post_setup):
+    with pytest.raises(Exception) as e:
+        get_reaction_metrics(post_id=9)
+    assert "Post Does not Exist" in str(e.value)
+
+
+@pytest.mark.django_db
+def test_post_reaction_metrics_metrics_data(post_setup, user_setup):
```

Comment:
assert len of res so that it will ensure only 3 reactions are returned

Reply:
Done